recipe/meta.yaml

```diff
@@ -97,21 +95,18 @@ outputs:
         - python
         - scipy
         - anndata >=0.10.1
-        - tiledb-py >=0.32.0,<0.33.0
+        - tiledb-py >=0.33.0,<0.34.0
```

Comment:
TileDB-Py is an optional dependency now, right? I recommend moving this requirement to the test section. That way we can still confirm it can be co-installed with the correct version of tiledb-py, but allow users the possibility to install a minimal tiledbsoma-py conda env.

Reply:
Thanks @jdblischak !

Let me 'get this green' for pre-check & unblock some things in the [dependency chain](https://github.com/single-cell-data/TileDB-SOMA/wiki/Branches-and-releases) tonight & I'll definitely do that moving forward